pkgs/development/python-modules/swspotify/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, python3 }:
+
+buildPythonPackage rec {
+  pname = "SwSpotify";
+  version = "1.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mrrg3zcm2ppxxzcva8nbyj1dkcplif4a88mvapphfd91a1d521q";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    requests flask-cors dbus-python
+  ];
+  
+  preConfigure = ''
+    substituteInPlace setup.py \
+      --replace 'requests>=2.24.0' 'requests~=2.23.0'
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/SwagLyrics/SwSpotify";
+    description = "Cross-platform library to get the currently playing song and artist from Spotify without using the API or the internet";
```

usually description is a small statement
```suggestion
    description = "Library to get the currently playing song and artist from Spotify";
```